### DISCUSSION TITLE
怎么平滑从sealer转到sealos

### DISCUSSION THREAD

**Discussion by geminivlpq** [labring/sealos#3797] [Q&A]
一开始用的sealer，想转用sealos，有什么方式能平滑切换，不用重新新建集群？

**Comment by omaidb** [comment]
> 一开始用的sealer，想转用sealos，有什么方式能平滑切换，不用重新新建集群？

同问，但是sealos要收费，你考虑清楚了

**Comment by fanux** [comment]
Sealos 安装 k8s 的功能并不收费，只用到 sealos 二进制。 sealos cloud 收费。 如果只是装 k8s 用 sealer 就好。新集群可以再考虑用 sealos
